src/libs/Pusher/pusher.js

```diff
@@ -331,19 +354,6 @@ function registerCustomAuthorizer(authorizer) {
     customAuthorizer = authorizer;
 }
 
-/**
- * Disconnect from Pusher
- */
-function disconnect() {
```

Comment:
No longer needed because Pusher will just be always connected, even if on public screens. cc @Julesssss because we would need to add this back if we are still going to bind the pusher connection to the foreground status of the mobile app (i.e: disconnect Pusher when the app is backgrounded).